lib/elixir_analyzer/test_suite/top_secret.ex

```diff
@@ -3,9 +3,11 @@ defmodule ElixirAnalyzer.TestSuite.TopSecret do
   This is an exercise analyzer extension module for the concept exercise Top Secret
   """
 
-  use ElixirAnalyzer.ExerciseTest
   alias ElixirAnalyzer.Constants
 
+  use ElixirAnalyzer.ExerciseTest,
+    suppress_tests: [Constants.solution_use_function_capture()]
```

Comment:
I wouldn't mind just removing this specialized top-secret check in favor of the custom one